src/main/java/uk/gov/digital/ho/hocs/casework/domain/repository/ActionDataExternalInterestRepository.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.digital.ho.hocs.casework.domain.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import uk.gov.digital.ho.hocs.casework.domain.model.ActionDataAppeal;
+import uk.gov.digital.ho.hocs.casework.domain.model.ActionDataDeadlineExtension;
+import uk.gov.digital.ho.hocs.casework.domain.model.ActionDataExternalInterest;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface ActionDataExternalInterestRepository extends CrudRepository <ActionDataExternalInterest, UUID> {
```

Comment:
I know I think I left some in in the last review... but the methods that aren't used: `findAllByCaseDataType`, `findByCaseDataUuid` and `findAllByCaseTypeActionUuid`, can you remove them please?